native/build.gradle

```
@@ -37,7 +37,7 @@ checkstyle {
     configProperties = ["suppressionFile" : file("${rootDir}/build-config/checkstyle/build/suppressions.xml")]
 }
 
-checkstyleMain.dependsOn(":build-config:checkstyle:downloadMultipleFiles")
+checkstyleMain.dependsOn(":checkstyle:downloadMultipleFiles")
```

Shall we rename this task (`downloadMultipleFiles`) to `downloadCheckstyleRuleFiles` as discussed? Have to update this is a few places.

```suggestion
checkstyleMain.dependsOn(":checkstyle: downloadCheckstyleRuleFiles")
```